client/state/sites/constants.js

```diff
@@ -71,3 +71,20 @@ export const SITE_REQUEST_OPTIONS = [
 	'difm_lite_site_options',
 	'site_intent',
 ].join();
+
+// Performance-optimized request for lists of sites.
+// Don't add more fields because you will make the request slower.
+export const SITE_EXCERPT_REQUEST_FIELDS = [
```

Comment:
Are we going to be using all of those fields? Could we reduce the list if there are unused fields?